addons/mass_mailing/views/assets.xml

```diff
@@ -6,7 +6,6 @@
             <link rel="stylesheet" type="text/scss" href="/mass_mailing/static/src/scss/mass_mailing_mobile.scss"/>
             <link rel="stylesheet" href="/mass_mailing/static/src/css/email_template.css"/>
 
-            <script type="text/javascript" src="/mass_mailing/static/src/js/mass_mailing.js"></script>
```

Comment:
I think that we forget to delete the file. Anyway, do we still want to do what?

Reply:
@adr-odoo I think this file is not needed anymore so I removed it in separate commit, what do you want here from me?